docs/syntax-reference.rst

```diff
@@ -1122,6 +1122,9 @@ Aggregate functions
 +========================+===========================+=====================================================================+
 | COUNT                  | ``COUNT(col1)``           |  Count the number of rows                                           |
 +------------------------+---------------------------+---------------------------------------------------------------------+
+| HISTOGRAM              | ``HISTOGRAM(col1)``       | Returns a map containing the distinct values of ``col1`` mapped to  |
+|                        |							             | the number of times each one occurs in the given window.            |
++------------------------+---------------------------+---------------------------------------------------------------------+
```

Comment:
Does this need a better formatting?

Reply:
very strange - it looks just fine in an editor on my local branch. will try pushing a spurious change to fix it up